NetCoreRestApi/ServiceLayer/Converters/ProductConverter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using BusinessLayer.Models;
+using ServiceLayer.DataTransferObjects;
+using System.Collections.Generic;
+
+namespace ServiceLayer.Converters
+{
+    public class ProductConverter
+    {
+        public ProductDTO ConverToDTO(ProductModel productModel)
+        {            
+            if (productModel == null) return null;
+            productModel.CategoryList ??= new List<CategoryModel>();
```

Comment:
move initialization to ProductModel class

Reply:
Initialization needs only if product Model.Category List is null, for prevention throwing null reference exeption

Reply:
Done